webrtc.html

```diff
@@ -11325,31 +11325,39 @@ <h2>
                 <dfn data-idl="">setCodecPreferences</dfn>
               </dt>
               <dd>
-		<div id="setcodecpreferences-receive">
+    <div id="setcodecpreferences-send-and-receive">
                 <p>
                   The {{setCodecPreferences}} method overrides the default
-                  receive codec preferences used by the <a>user agent</a>. When
+                  codec preferences used by the <a>user agent</a>. When
                   generating a session description using either
                   {{RTCPeerConnection/createOffer}} or
-                  {{RTCPeerConnection/createAnswer}}, the <a>user agent</a>
-                  MUST use the indicated codecs, in the order specified in the
-                  <var>codecs</var> argument, for the media section
-                  corresponding to this {{RTCRtpTransceiver}}.
+                  {{RTCPeerConnection/createAnswer}}, the <a>user agent</a> MUST
+                  filter the preferred codecs on {{RTCRtpTransceiver/direction}}
+                  and, if this results in a non-empty list, it MUST use the
+                  specified codecs in the order of the <var>codecs</var>
+                  argument, for the media section corresponding to this
+                  {{RTCRtpTransceiver}}.
                 </p>
                 <p>
                   This method allows applications to disable the negotiation of
                   specific codecs (including RTX/RED/FEC). It also allows an
-                  application to cause a remote peer to prefer the codec that
-                  appears first in the list for sending.
+                  application to affect the remote peer's default codecs;
+                  influencing both what they should send by default and the
+                  default codec order of subsequently generated remote SDP. But
+                  just like the local peer can override its defaults, so can the
+                  remote peer. The local peer MUST be prepared to receive any
+                  codec that has been negotiated, even ones not first in the
+                  list.
```

Comment:
We should avoid competing normative language with JSEP. One way might be to put explanations in non-normative notes.

Reply:
I changed these to notes and tried to be more clear